Documentation/README.md

```diff
@@ -0,0 +1,17 @@
+
+# Documentation
+
+-   [How to use the API][API]
+
+<br/>
+
+## Commands
+
+-   [How to use the **Spaced** command][Spaced]
+
+-   [How to use the **Radial** command][Radial]
+
+
+[Spaced]: ./Commands/Spaced
+[Radial]: ./Commands/Radial
+[API]: ./API
```

Comment:
These three need to end in .md in order for the links to work in Github I think. 

When I tried to click on the links in this page I get 404s.